### PR TITLE
Added HSL to RGB and RGB to HSL snippets. 2 commits only.

### PR DESCRIPTION
**Pull request by soham0-0**

### CODE REVIEW

**Comment by soham0-0** [comment]
@Trinityyi I have made the changes. Please have a look.

File: snippets/HSLToRGB.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: HSLToRGB
+tags: math,intermediate
+---
+
+Converts a HSL color tuple to RGB format.
+
+- Use the [HSL to RGB conversion formula](https://en.wikipedia.org/wiki/HSL_and_HSV#HSL_to_RGB) to convert to the appropriate format.
+- The range of the input parameters is H: [0, 360], S: [0, 100], L: [0, 100].
+- The range of all output values is [0, 255].
```
Comment (Stefalb89) [inline snippets/HSLToRGB.md:10]: //

Comment (soham0-0) [reply snippets/HSLToRGB.md:10]: I didn't understand that. Could you please clarify?

**Comment by davoodgf** [comment]
در خدمتم

**Comment by davoodgf** [comment]
در خدمتم بفرمائید

**Comment by soham0-0** [comment]
@Chalarangelo @Trinityyi There has been some updates in Hacktoberfest so could you please follow these guidelines so that my PRs count?
thank you so much!
[Update link](https://hacktoberfest.digitalocean.com/hacktoberfest-update)

**Review by Chalarangelo — APPROVED** [review]
Thanks for contributing. I'll make a few minor changes and merge this.

**Pull request merged**